explainer.md

```diff
@@ -0,0 +1,115 @@
+# Autoplay Detection API
+
+## Problem
+If a website plays a video it cannot detect whether it will be allowed to autoplay or whether that autoplay will be blocked until it calls play on the media element.
+
+## Goal
+A site should be able to determine whether a video will autoplay or not. Knowing this in advance is important for a site as they may want to adjust their user experience or select alternate content to use instead.
+
+## Proposed API Design
+The document has an autoplay policy which can change. There will be an API on the document to get the autoplay policy and another on the media element which will return a boolean as to whether the media will be able to autoplay.
+
+There are three types of autoplay policy:
+
+ * `allowed` - autoplay is allowed
+ * `allowed-muted` - muted autoplay is allowed
+ * `disallowed` - document level autoplay is not allowed. The site should check on the element whether element level autoplay is allowed
+
+The API is defined as the following:
+
+```javascript
+enum AutoplayPolicy {
+  “allowed”,
```

Comment:
(Nit-pick) These “curly quotes” should be replaced by "regular double quotes", to make it valid JavaScript.